src/components/ViewedByCustomer/ViewedByCustomer.scss

```diff
@@ -0,0 +1,18 @@
+@import '../../sass/Variables.scss';
+@import '../../sass/Text.scss';
+
+.CoveoViewedByCustomer {
+    color: black;
+    font-weight: bold;
+    font-size: 14px;
```

Comment:
why 14px ?
Why not 1em ?

Reply:
We don't use em to fix font size in the search-ui & co.